### PR TITLE
Web Components V3 - Popover dev spec

### PR DESCRIPTION
**Pull request by bearcat-msft**
# Popover

A popover is a small surface that appears when someone interacts with a component to give nonessential, contextual information without blocking them.

Popovers can have structured content and interactive components. If you need to show a more complex layout or block the page behind, try a dialog. For unstructured plain text, try a tooltip.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/12da47698c9121c0345c2f0db481cbae7a91b66e'>12da47698c9121c0345c2f0db481cbae7a91b66e</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

File: packages/web-components/src/popover/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+\*[Positioning Shorthand](https://react.fluentui.dev/?path=/docs/concepts-developer-positioning-components--default#shorthand-positions)
+
+### type Placement (floating-ui) vs PositionShorthand (FluentUI)
+
+The popover web component uses floating-ui (a javascript npm package) to implement the design guidelines. FluentUI Design guidelines have two concepts in regard to positioning:
+
+1. Position - "above" | "below" | "before" | "after"
+2. Alignment - "start" | "end" | "center"
+
+Below is a maping of the floating ui Placement type to the FluentUI PositioningShorthand type. There are some differences in language, but the meanings in the ui are the same. For the sake of clarity the web component implementation precisely matches the language of the react implementation:
```
Comment (procload) [inline packages/web-components/src/popover/README.md:65]: Do you anticipate taking a dependency on FloatingUI for the positioning logic or do you plan to implement this on your own?

**Review by procload — COMMENTED** [review]
A couple other questions:
* Maybe we should include the proposed markup for the template here to indicate what type of element we'll be using. Is it a `dialog`?
* What sort of ARIA labels do we need to put on this? Do we need `aria-describedby` tied to its trigger element. Will this run into cross-root ARIA issues?

File: packages/web-components/src/popover/README.md (hunk 1 above)
Comment (bearcat-msft) [reply packages/web-components/src/popover/README.md:96]: We will be using the popover api and css anchor positioning if possible.

**Comment by Hotell** [comment]
looks like bad merge/rebase ? 

would you mind removing reviewers that are not intended to review this PR ? ty

image

**Comment by bearcat-msft** [comment]
> looks like bad merge/rebase ?
> 
> would you mind removing reviewers that are not intended to review this PR ? ty
> 
> image

No problem, thanks for the heads up.

**Comment by JustSlone** [comment]
@bearcat-msft closing as that is the only way to stop notifying reviewers. Please update your PR to point to master following this guide: https://github.com/microsoft/fluentui/pull/31361 and reopen when it's updated. Thanks!

**Pull request closed without merging**